src/system/Wizard/Wizard.js

```diff
@@ -14,7 +14,25 @@ import { MdArrowForward } from 'react-icons/md';
 import { Box, WizardStep, Flex, WizardStepHorizontal } from '..';
 
 const Wizard = React.forwardRef(
-	( { steps, activeStep, variant, completed = [], className = null, ...props }, forwardRef ) => {
+	(
+		{
+			steps,
+			activeStep,
+			variant,
+			completed = [],
+			className = null,
+			titleAutofocus = false,
+			...props
+		},
+		forwardRef
+	) => {
+		const [ initialStep, setInitialStep ] = React.useState( activeStep );
+		useLayoutEffect( () => {
+			if ( titleAutofocus && activeStep !== initialStep ) {
```

Comment:
I forgot to submit this comment a few days ago, apologies.

Reply:
@kat3samsin Maybe I should rename it. So, this is the initial status of the wizard. The idea is that, when the wizard is first shown, we shouldn't focus on the title. This is because otherwise we might grab the focus from other elements in the page. 
We want the focus to be controlled when moving inside the wizard. Or at least that's how I envisioned.

Reply:
Got it, I see what you mean. I wonder, though if we should direct users to the current step if for example they are in the middle of launching [site](https://dashboard.wpvip.com/apps/6618/production/network-sites/launch/5), I needed to navigate to the previous steps before I could go to the current step. What do you think?

Reply:
I now understand what you  meant. The issue is that I use activestep as the starting point where I should have some firstLoad state. Will change that logic so that it works if we start the wizard on a different step

Reply:
> I needed to navigate to the previous steps before I could go to the current step. What do you think?

I think this is accessible in my view, but I guess this is debatable.
I normally scan the other headers to have an understanding of the status, that's why I don't think it's a bad idea to -not- focus on first render. 

An alternative to this would be to make the `titleAutofocus` be an enum (or not a boolean) so that we can require different behaviours. 

In any case, if you pass the title not as a string, but as a component the behaviour can be entirely customized if needed.
The `titleAutofocus` kicks in only to make the "default" more accessible.

Inspired by the comment I also removed the dependency on `activeStep` to track the first render and used a more generic approach here https://github.com/Automattic/vip-design-system/pull/175/commits/c25680c49f8289f80fc3657db39493fb8f311481

Reply:
> I think this is accessible in my view, but I guess this is debatable.
Yes, I agree this depends on the use case of the wizard.

>Inspired by the comment I also removed the dependency on activeStep to track the first render and used a more generic approach here https://github.com/Automattic/vip-design-system/commit/c25680c49f8289f80fc3657db39493fb8f311481
Thanks for updating!